symbolic-symcache/tests/test_cache.rs

```diff
@@ -50,7 +50,7 @@ fn test_load_header_macos() -> Result<(), Error> {
         functions: 639,
         source_locations: 6033,
         ranges: 4591,
-        string_bytes: 42829,
+        string_bytes: 43020,
```

Comment:
interesting observation; we have a tiny increase in `string_bytes`, but the symcache (fixture) itself is smaller.

I’m also surprised we have the same number of functions. Which snapshot test had the duplicated functions previously? Can we check that specific test if we drop those duplicated functions now?